### PR TITLE
refactor(autoware_pointcloud_preprocessor): rework crop box parameters

### PR DESCRIPTION
**Pull request by vividf**
## Description
This PR includes the following changes
1. Remove the default parameters for the crop box filter node. 
2. Add parameter, launch file, and parameters schema for the node. 
3. Change the name `nodelet` to `node`.

A big thank you to @Ariiees for the PRs (https://github.com/autowarefoundation/autoware.universe/pull/8297 and https://github.com/autowarefoundation/autoware.universe/pull/7422), where we worked together on the entire pointcloud preprocessor parameters, launch file, and schema.

Note that the schema for filter.param.yaml will be implemented in another PR. 

## Related links

**Parent Issue:**

- Link

## How was this PR tested?
`ros2 launch autoware_pointcloud_preprocessor crop_box_filter_node.launch.xml`

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by knzo25 — COMMENTED** [review]
Should `filter.param.yaml` be in this PR?

**Comment by vividf** [comment]
@knzo25 It is not needed in this PR, I put it here just because when the launch file launch without having the filter.param.yaml file, it will show a warning message. (but it will not cause any issue since the filter.cpp has the default parameter, and that's why I remove the filter.param.yaml in other PRs). I will also remove this one. Thanks!

**Comment by vividf** [comment]
@knzo25  kindly ping

**Comment by vividf** [comment]
@badai-nguyen @knzo25 @drwnz

**Review by badai-nguyen — CHANGES_REQUESTED** [review]
@vividf Please add missing parameter (`negative: false`) for [this](https://github.com/autowarefoundation/autoware.universe/blob/f31e5dc592626af392ef0321b75b031b132537b2/perception/autoware_euclidean_cluster/config/voxel_grid_based_euclidean_cluster.param.yaml#L17) since crop_box_filter also used in euclidean_cluster pipeline.
I created PR for neccessary modification in [autoware_launch](https://github.com/autowarefoundation/autoware_launch/pull/1155) and please wait until it is merged.

File: sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json
Comment (knzo25) [inline sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json:11]:
```suggestion
          "description": "minimum x-coordinate value for crop range",
```

Comment (knzo25) [reply sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json:11]: Applicable to other instances. Units

File: sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json
Comment (knzo25) [inline sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json:41]: write this one better please

**Review by knzo25 — CHANGES_REQUESTED** [review]
Documentation related comments

**Comment by vividf** [comment]
@badai-nguyen thanks for the check, added in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/add0cb7a341610c852b3fc33eff14485a790bb09

File: sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json
Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json]: Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/772c90251b465caeab41535c50fc12f39ab10a32

File: sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json
Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json]: Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/772c90251b465caeab41535c50fc12f39ab10a32

File: sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json
Comment (knzo25) [inline sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json:41]: Using the work `filter` is ambiguous, even more due to how this parameter is called. Use filter in or filter out (I have already made this comment before)

Comment (vividf) [reply sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json]: Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/7f917b88277714dffb5aee82a718900bfc5caa50

**Review by knzo25 — APPROVED** [review]
LGTM

**Review by badai-nguyen — APPROVED** [review]
LGTM

**Pull request merged**
